### ISSUE TITLE
Hide Top?

### ISSUE THREAD

**Issue by Dontmindmes** [asticode/go-astilectron#325]
Hey is it possible to hide this?
https://i.imgur.com/YaP4hE1.png

**Comment by asticode** [comment]
I believe you need to set [this attribute](https://github.com/asticode/go-astilectron/blob/master/window.go#L97) to `false`.

Let me know how it goes

**Comment by CarsonSlovoka** [comment]
tl;dr

try also [Frame attribute](https://github.com/asticode/go-astilectron/blob/18cf1d532adc9860ff3c11df738a896127d203de/window.go#L101) to false

----

Hi @Dontmindmes 

If you want to hide the entire title bar, try to

> `new BrowserWindow({frame: false})`

> see [new BrowserWindow([options])](https://www.electronjs.org/docs/latest/api/browser-window#new-browserwindowoptions) <sup>[permalink](https://github.com/electron/electron/blob/37a904c299c4d7bde4b2b5007a52043bca383868/docs/api/browser-window.md?plain=1#L187-L188)</sup>

> frame boolean (optional) - Specify false to create a [frameless window](https://www.electronjs.org/docs/latest/tutorial/window-customization#create-frameless-windows). Default is true.

### details

```go
	if w, err = a.NewWindow("index.html", // it will call atilectron.index.js.consts.eventNames.windowCmdCreate
		&astilectron.WindowOptions{  // 👈 json.windowOptions  see explain below
			Center: astikit.BoolPtr(true),
			Height: astikit.IntPtr(700),
			Width:  astikit.IntPtr(700),
			// Closable:       astikit.BoolPtr(false),
			Frame: astikit.BoolPtr(false),   // 👈 add this
		}); err != nil {
		l.Fatal(fmt.Errorf("main: new window failed: %w", err))
	}
```

1. [ atilectron.index.js.consts.eventNames.windowCmdCreate  -> windowCreate(json) ](https://github.com/asticode/astilectron/blob/a8c3f044b1feecab7f18840f6769ed7ec576da1d/index.js#L232-L233) 
2. [windowCreate(json) -> new BrowserWindow(json.windowOptions) ](https://github.com/asticode/astilectron/blob/a8c3f044b1feecab7f18840f6769ed7ec576da1d/index.js#L413)  for example `new BrowserWindow({frame: false})`
3. json.windowOptions define in the [window.go : WindowOptions](https://github.com/asticode/go-astilectron/blob/master/window.go#L101)
4. good thing is @asticode already define for us, so you can set `a.NewWindow("", &astilectron.WindowOptions{Frame: astikit.BoolPtr(false)},);` done!

## see also 
- [Hide titlebar on window hover in electron?](https://stackoverflow.com/a/50584474/9935654)
